### PR TITLE
change sw padding mode

### PR DESCRIPTION
**Pull request by heyufan1995**
Fixes # .

### Description
A few sentences describing the changes proposed in this pull request.

### Status
**Ready/Work in progress/Hold**

### Please ensure all the checkboxes:

- [x] Codeformat tests passed locally by running `./runtests.sh --codeformat`.
- [ ] In-line docstrings updated.
- [ ] Update `version` and `changelog` in `metadata.json` if changing an existing bundle.
- [ ] Please ensure the naming rules in config files meet our requirements (please refer to: `CONTRIBUTING.md`).
- [ ] Ensure versions of packages such as `monai`, `pytorch` and `numpy` are correct in `metadata.json`.
- [ ] Descriptions should be consistent with the content, such as `eval_metrics` of the provided weights and TorchScript modules.
- [ ] Files larger than 25MB are excluded and replaced by providing download links in `large_file.yml`.
- [ ] Avoid using path that contains personal information within config files (such as use `/home/your_name/` for `"bundle_root"`).

### CODE REVIEW

File: models/vista3d/scripts/inferer.py [hunk 1]
```diff
@@ -101,7 +101,7 @@ def __call__(
             )
         else:
             val_outputs = SlidingWindowInfererAdapt(
-                roi_size=self.roi_size, sw_batch_size=self.sw_batch_size, with_coord=True
+                roi_size=self.roi_size, sw_batch_size=self.sw_batch_size, with_coord=True, padding_mode='replicate'
```
Comment (bhashemian) [inline models/vista3d/scripts/inferer.py]:
```suggestion
                roi_size=self.roi_size, sw_batch_size=self.sw_batch_size, with_coord=True, padding_mode="replicate"
```

**Review by bhashemian — APPROVED** [review]
LGTM!

File: models/vista3d/configs/metadata.json [hunk 2]
```diff
@@ -2,6 +2,7 @@
     "schema": "https://github.com/Project-MONAI/MONAI-extra-test-data/releases/download/0.8.1/meta_schema_20240725.json",
     "version": "0.5.6",
```
Comment (yiheng-wang-nv) [inline models/vista3d/configs/metadata.json]: Please also modify the version

**Pull request merged**